composer.json

```diff
@@ -40,7 +40,7 @@
         }
     ],
     "require-dev": {
-        "phpunit/phpunit": "^4.8",
+        "phpunit/phpunit": "^4.8|^5.7",
```

Comment:
Instead of 5.7, can you add both PHPUnit 6 and PHPUnit 7?

- PHPUnit 4 will cover 5.4 - 5.6
- PHPUnit 5 will cover 7.0
- PHPUnit 6 will cover 7.1 - 7.3

See https://phpunit.de/supported-versions.html

This should also be a separate commit.